src/pages/en/guides/deploy/github.md

```diff
@@ -15,15 +15,27 @@ Astro maintains the official `withastro/action` to deploy your project with very
 
 1. Set the [`site`](/en/reference/configuration-reference/#site) and, if needed, [`base`](/en/reference/configuration-reference/#base) options in `astro.config.mjs`.
     - `site` should be something like `https://<YOUR_USERNAME>.github.io`
```

Comment:
I know this was existing text when you got here, but "should be something like" without more explicit instruction makes me nervous. 😅 

I think I recall from earlier versions of this page that you SHOULD be using `https://<YOUR_USERNAME>.github.io` -- but, is it possible to deploy TWO DIFFERENT sites on GitHub pages? What would you do then, since you only have one username.

I think it might also be helpful to be more DIRECT about saying to use this for `site` (like, just use this!) or to add what another "something like" could be, for example in the case of having two sites deployed. (Is that even possible?)

Reply:
Multiple sites are possible, and as far as I can tell, the `site` for them is the same. `sarah11918.github.io/blog` and `sarah11918.github.io/portfolio` both share a site of `sarah11918.github.io`.

The other "something like" is when you use a custom domain. Then it's the custom domain you fill in as `site`.

Reply:
Ah, good call, that's where base comes in !

I thought about also suggesting that we show a custom domain as a possiblility there, but then our tip for "Want to use a custom domain?" is alllll the way down at the bottom. And, THAT doesn't say to update SITE, so I didn't know whether that cname file would actually just "overwrite" a github.io site, or whether you were supposed to update SITE, too.

So then, what about something like:

```
- `site` should be `https://<YOUR_USERNAME>.github.io` or `https://my-custom-domain.com`
```

Reply:
You are right, we forgot to include an update site in that little tip 😅. my best guess is that this file sets an alias for `username.github.io` but the build is using site, whoops 😇

Reply:
```suggestion
    - `site` be should be `https://<YOUR_USERNAME>.github.io` or `https://my-custom-domain.com`
```